homeassistant/components/myuplink/entity.py

```diff
@@ -26,3 +26,26 @@ def __init__(
         # Basic values
         self._attr_unique_id = f"{device_id}-{unique_id_suffix}"
         self._attr_device_info = DeviceInfo(identifiers={(DOMAIN, device_id)})
+
+
+class MyUplinkSystemEntity(CoordinatorEntity[MyUplinkDataCoordinator]):
```

Comment:
Why doesn't this inherit the class above and overwrite the relevant properties?

Reply:
Done!